docs/accelerators.md

```diff
@@ -49,7 +54,7 @@ docker run --rm -it  --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/rende
 GPU device can be used also on Windows hosts with Windows Subsystem for Linux 2 (WSL2). In such scenario, there are needed extra docker parameters. See the command below.
 Use device `/dev/dxg` instead of `/dev/dri` and mount the volume `/usr/lib/wsl`:
 
-```bash
+```
```

Comment:
So this part is not supposed to be executed?

Reply:
it's for wsl. automatic tests do not support it